docs/pages/versions/unversioned/sdk/screen-capture.mdx

```diff
@@ -54,26 +54,35 @@ export default function ScreenCaptureExample() {
 <SnackInline label="Screen Capture functions" dependencies={["expo-screen-capture", "expo-media-library"]}>
 
 ```js
```

Comment:
````suggestion
```tsx
````

Out of the scope of this PR but since subscription types are added, let's change the file extension here for accurate syntax highlighting.

Reply:
I decided to remove the type, because the "open in snack" button opens a snack with App.js file